apps/embed/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^$',
+        views.EmbedView.as_view(), name='idea-embed'),
```

Comment:
I don't think we should call it idea-embed. For simplicity we can just call it embed and change afterwards.